src/Digdir.Domain.Dialogporten.Application/Features/V1/EndUser/Dialogs/Queries/Search/SearchDialogQuery.cs

```diff
@@ -172,12 +173,13 @@ public async Task<SearchDialogResult> Handle(SearchDialogQuery request, Cancella
             )
             .Where(x => !x.VisibleFrom.HasValue || _clock.UtcNowOffset > x.VisibleFrom)
             .Where(x => !x.ExpiresAt.HasValue || x.ExpiresAt > _clock.UtcNowOffset)
-            .ProjectTo<SearchDialogDto>(_mapper.ConfigurationProvider)
             .ToPaginatedListAsync(request, cancellationToken: cancellationToken);
 
-        await FetchRelevantActivities(paginatedList, userPid, cancellationToken);
+        var paginatedDtoList = paginatedList.To(_mapper.Map<SearchDialogDto>);
```

Comment:
Er det et alternativ å fremdeles bruke ProjectTo, men endre i MappingProfile til 

```
CreateMap<DialogEntity, SearchDialogDto>()
            .ForMember(dest => dest.LatestActivities, opt => opt.Ignore())
```

Da trenger man vel ikke den `Include()`-en for `Content`, ei heller den `To`-utility-en?

Reply:
Da drar vi med oss alle aktiviteter for alle dialoger i lista
Det var vel det @MagnusSandgren ville unngå, unødig db-trafikk

Reply:
Ah, vent. 
Du mener ignorere den i mappinga, bruke ProjectTo, og så hente activities og sette de manuelt?

Reply:
Funker fint det, @elsand 👍🏼